lib/edit-site-page.php

```diff
@@ -107,7 +107,7 @@ function gutenberg_edit_site_init( $hook ) {
 		$max_upload_size = 0;
 	}
 
-	// This filter is documented in wp-admin/includes/media.php.
+	/** This filter is documented in wp-admin/includes/media.php */
```

Comment:
It looks like this comment is no longer present in the codebase.

Reply:
Thanks, my local was waaay out of date.